x-pack/test/functional/apps/monitoring/feature_controls/monitoring_spaces.ts

```diff
@@ -21,9 +21,8 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
     });
 
     after(async () => {
-      await esArchiver.unload('x-pack/test/functional/es_archives/empty_kibana');
-      await PageObjects.common.navigateToApp('home');
```

Comment:
__note:__ I cannot think of any valid use case when we'd need to go to `home` and then immediately logout. Let's see if CI agrees.